guide/17-geocoding-guides/part1_what_is_geocoding.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<p>-&nbsp;Search for places: A user can use geocode() to&nbsp;find the locations of geographic places around the world,&nbsp;locate businesses near a location, search for places by category, such as restaurants, gas stations, or schools (The Geocoding Service allows you to search for different categories of places, such as businesses, restaurants, geographic features, and administrative areas), or find and display places on a map. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>